reviewguide/review_points_guide.md

```diff
@@ -0,0 +1,221 @@
+# Review points in Guide
+
+This page describes the review points that were often asked to be updated.
+
+When you review a guide page, check these points and get the page consistency.
+
+- [Headings](#headings)
+- [Created page](#create)
+   - [toc_all.md](#create_toc_all)
+   - [overview.md](#create_overview)
+- [Changed file name / delete the page](#change_delete)
+- [Code block](#codeblock)
+   - [Indentation of the code block](#codeblock_inden)
+   - [Code block type](#codeblock_type)
+   - [Consistency between code block and description](#codeblock_check)
+- [Hyper link](#hyperlink)
+   - [Page in docs.tizen.org](#hyperlink_guide)
+   - [Native or Web API](#hyperlink_not_dotnet)
+   - [.NET API](#hyperlink_dotnet)
+   - [Outside of docs.tizen.org](#hyperlink_out)
+- [Images](#images)
+- [Related information](#related_info)
+   - [Dependencies](#related_info_dependencies)
+
+
+
+<a name="headings"></a>
+
+## Headings
+
+- Follow the sentence case :
+
+  **Ref. :** https://github.com/Samsung/tizen-docs/pull/1501#discussion_r723023920
+
+- The heading uses verb or noun, but not gerund :
+
+   **Example :** Use "Check points" instead of "Checking points".
+   
+- After the heading, simple sentences to describe the basic knowledge are recommended.
+
+   ![Simple description](./media/guide_simple_description.png)
+
+<a name="create"></a>
+
+## Created page

```

Comment:
Can we say "Add new topic" "Add new page"

Reply:
Done